packages/seo-store/src/editor/slice.js

```diff
@@ -13,12 +11,38 @@ const initialState = {
 };
 
 const editorSlice = createSlice( {
-	name: EDITOR_SLICE_NAME,
+	name: "editor",
 	initialState,
-	reducers: createSimpleReducers( EDITOR_SLICE_NAME, Object.keys( initialState ) ),
+	reducers: {
+		updateContent: ( state, action ) => {
+			state.content = action.payload;
+		},
+		updateTitle: ( state, action ) => {
```

Comment:
```suggestion
		updateEditorTitle: ( state, action ) => {
```

Reply:
Let's keep this `title` unprefixed, since it is the actual title. If we run into problems later we can prefix then.